### ISSUE TITLE
Multiple Mask Tokens

### ISSUE THREAD

**Issue by zbush548** [huggingface/transformers#5641]
For those wishing to [MASK] several tokens, here this is.

My question, however, relates to the output. I added "top_k" assuming I'd be able to return multiple sentences, but that was not the case. I am not sure how exactly I can achieve this. 

```
import torch
from transformers import BertTokenizer, BertModel,BertForMaskedLM
 
tokenizer = BertTokenizer.from_pretrained('bert-base-cased')
 
input_tx = "[CLS] [MASK] [MASK] [MASK] of the United States mismangement of the Coronavirus is its distrust of science. [SEP]"
tokenized_text = tokenizer.tokenize(input_tx)
indexed_tokens = tokenizer.convert_tokens_to_ids(tokenized_text)
top_k = 10
tokens_tensor = torch.tensor([indexed_tokens])
segments_tensors = torch.tensor([[0]*25])
 
model = BertForMaskedLM.from_pretrained('bert-base-cased')
 
outputs = model(tokens_tensor, token_type_ids=segments_tensors)
predictions = outputs[0]
predicted_index = [torch.argmax(predictions[0, i]).item() for i in range(0,24)]
predicted_token = [tokenizer.convert_ids_to_tokens([predicted_index[x]])[0] for x in range(1,24)]
print(predicted_token)

```

`Output: 'The', 'main', 'cause', 'of', 'the', 'United', 'States', 'mi', '##sman', '##gement', 'of', 'the', 'Co', '##rona', '##virus', 'is', 'its', 'di', '##st', '##rust', 'of', 'science', '`

**Comment by LysandreJik** [comment]
Hi! This is a very good question. We just opened a forum on discuss.huggingface.co to discuss those kind of questions exactly. Do you think you could go over there and ask it? Thanks a lot!

**Comment by zbush548** [comment]
Sure, I'll do that now! @LysandreJik
